File: src/main/java/my/maryvk/maryvkweb/web/SeekerStatus.java
package my.maryvk.maryvkweb.web;

import lombok.Data;

@Data
public class SeekerStatus {
    private final int userId;
    private final Status status;

    public SeekerStatus(int userId, boolean isRunning) {
        this.userId = userId;
        this.status = isRunning ? Status.RUNNING : Status.STOPPED;
    }

    public String changeStateStr() {
        return status.equals(Status.RUNNING) ? "stop" : "start";
    }

    public enum Status {
        RUNNING,
        STOPPED
    }
}

File: src/main/java/my/maryvk/maryvkweb/web/SeekerStatus.kt
package my.maryvk.maryvkweb.web

data class SeekerStatus(val userId: Int, val isRunning: Boolean) {
    val status: Status

    init {
        this.status = if (isRunning) Status.RUNNING else Status.STOPPED
    }

    fun changeStateStr(): String {
        return if (status == Status.RUNNING) "stop" else "start"
    }

    enum class Status {
        RUNNING,
        STOPPED
    }
}